paper-input-error.html

```diff
@@ -65,20 +65,5 @@
     behaviors: [
       Polymer.PaperInputAddonBehavior
     ],
-
-    properties: {
-      /**
-       * True if the error is showing.
-       */
-      invalid: {
```

Comment:
I think this can be be cleaned up in a 2.0 release, but maybe we shouldn't merge this for now. Adding the appropriate label for tracking